index.js

```diff
@@ -186,6 +186,10 @@ var transforms = [
         .replace(/ (\d)/, '-$1')
         .replace(/ ?International/, '') +
       '-4.0'
+  },
+  // e.g. 'GNU (LESSER GENERAL PUBLIC LICENSE | LGPL)
+  function (argument) {
+    return argument.replace(/GNU (lesser general public license|lgpl).*/gi, 'LGPL-3.0-or-later')
```

Comment:
Did you find this `license` string in a package on the npm public registry?

Reply:
I have not seen anything on the npm public registry yet but have seen a few in pypi. Here are some examples:
- https://pypi.org/project/dominate/
- https://pypi.org/project/mdt/1.2.2/